TIL/topics/dart_basics/Collection.md

```
@@ -0,0 +1,41 @@
+    - 순서가 보장되지 않음
+    - entries Property를 통해서 forEach()를 사용할 수 있다.(다만 순서는 보장되지 않음)
+  - ### Set
+      ```dart
+        Set<int> peopleAges = {20, 24, 25};
+      ```
+    - 중복을 허용하지 않음
+    - get() 메서드를 제공하지 않음
+    - 반복문 필요시 iterator 혹은 forEach()를 사용
+    - search 시 contains() 메서드의 속도는 list보다 압도적으로 빠름
+   
+  - ### Collection KeyPoint
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)